firmware/src/widgets/weatherWidget.cpp

```diff
@@ -128,31 +129,21 @@ bool WeatherWidget::getWeatherData() {
 }
 
 void WeatherWidget::displayClock(int displayIndex) {
-    const int clockY = 94;
-    const int dayOfWeekY = 160;
-    const int dateY = 197;
+    const int clockY = 120;
+    const int dayOfWeekY = 190;
+    const int dateY = 50;
 
     m_manager.selectScreen(displayIndex);
+    m_manager.fillScreen(m_backgroundColor);
+    m_manager.setFontColor(m_foregroundColor);
 
-    TFT_eSPI &display = m_manager.getDisplay();
```

Comment:
In this case I don't think the manager should be used directly for drawing, but for future reference if you want to keep a diff smaller then something like this could be used so that all other lines don't need changing:
```cpp
auto& display = m_manager;
```
That allows to keep the code below doing this kind of stuff intact:
```cpp
    display.fillScreen(m_backgroundColor);
    display.setTextColor(m_foregroundColor);
    display.setTextSize(1);
```